frontend/src/admin/use-surveys.ts

```diff
@@ -40,11 +44,17 @@ export const useSurveys = (): UseSurveyReturn => {
         setSurveys(surveys.filter(survey => survey.id.toString() !== surveyId.toString()))
     }
 
+    const removeIndexSurvey = (id: string) => {
+        setIndexSurveys(indexSurveys.filter(survey => survey.id.toString() !== id))
+    }
+
     return {
         loading,
         surveys,
         changeSurvey,
         removeSurvey,
+        indexSurveys,
+        removeIndexSurvey
```

Comment:
This hook should now be just for IndexSurvey. The code for old surveys can be removed.

Reply:
Is done